src/figuresClasses.ts

```diff
@@ -1,11 +1,82 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: Function;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape = 'triangle',
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('Invalid parameters.');
+    }
+
+    const sides: number[] = [sideA, sideB, sideC].sort((a, b) => b - a);
+
+    if (sides[0] >= sides[1] + sides[2]) {
+      throw new Error(
```

Comment:
[CODE STYLE] This error message is good, but it could be even more descriptive by specifying that the sum of any two sides must be greater than the third side to form a valid triangle.